### PR TITLE
add API to load mitochondrial content

### PR DESCRIPTION
**Pull request by aerlaut**
# Background
#### Link to issue 

https://biomage.atlassian.net/browse/BIOMAGE-490

#### Link to staging deployment URL 

Staging deployment in related UI pull request : https://github.com/biomage-ltd/ui/pull/127

#### Links to any Pull Requests related to this

https://github.com/biomage-ltd/ui/pull/127

#### Anything else the reviewers should know about the changes here

- Added a new worker request and worker response schema to fetch cell metadata

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [X] Tested locally with Inframock
- [X] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

**Comment by aerlaut** [comment]
@marcellp @ivababukova I created a blanket WorkRequest and WorkReponse body to get cell metadata (e.g. mitochondrial content, doublet score) : `WorkRequestLoadCellMeta.v1.yaml` and `WorkResponseLoadCellMeta.v1.yaml`.

Is it ok to merge them like this? or better keep it separate, e.g. 1 YAML for mitochondrial content, another YAML for doublet score,

File: src/specs/models/WorkRequestBodies.v1.yaml [hunk 1]
```diff
@@ -6,3 +6,4 @@ oneOf:
   - $ref: ./work-request-bodies/WorkRequestDifferentialExpression.v1.yaml
   - $ref: ./work-request-bodies/WorkRequestListGenes.v1.yaml
   - $ref: ./work-request-bodies/WorkRequestGeneExpression.v1.yaml
+  - $ref: ./work-request-bodies/WorkRequestMitochondrialContent.v1.yaml
```
Comment (marcellp) [inline src/specs/models/WorkRequestBodies.v1.yaml:9]: This ref seems to point to something that does not exist?

**Comment by marcellp** [comment]
> @marcellp @ivababukova I created a blanket WorkRequest and WorkReponse body to get cell metadata (e.g. mitochondrial content, doublet score) : `WorkRequestLoadCellMeta.v1.yaml` and `WorkResponseLoadCellMeta.v1.yaml`.
> 
> Is it ok to merge them like this? or better keep it separate, e.g. 1 YAML for mitochondrial content, another YAML for doublet score,

I'm happy to go with whathever you think is better.

File: src/specs/models/WorkRequestBodies.v1.yaml (hunk 1 above)
Comment (aerlaut) [reply src/specs/models/WorkRequestBodies.v1.yaml:9]: I edited with Spotlight Studio and seemed forgot to press save. File added to replace `WorkRequestLoadCellMetadata`

**Comment by aerlaut** [comment]
> > @marcellp @ivababukova I created a blanket WorkRequest and WorkReponse body to get cell metadata (e.g. mitochondrial content, doublet score) : `WorkRequestLoadCellMeta.v1.yaml` and `WorkResponseLoadCellMeta.v1.yaml`.
> > Is it ok to merge them like this? or better keep it separate, e.g. 1 YAML for mitochondrial content, another YAML for doublet score,
> 
> I'm happy to go with whathever you think is better.

In the end I think decide to separate it, to keep the response / request flexible if there is a need to modify them in the future.

**Review by marcellp — APPROVED** [review]

**Pull request merged**